src/wallet/coincontrol.h

```diff
@@ -17,23 +17,23 @@ class CCoinControl
 {
 public:
     //! Custom change destination, if not set an address is generated
-    CTxDestination destChange;
+    CTxDestination destChange{CNoDestination()};
```

Comment:
If all local variables have a default initialized, you don't need the SetNull call in the constructor anymore.

Reply:
They don't -- specifically the new argument requires `gArgs` which are not available here.

I should probably drop "Declare-&-initialize..." commit.

Reply:
Dropped that commit.